projects/infineon/xmc4800_iotkit/dave4/aws_demos/.project

```diff
@@ -653,6 +653,11 @@
 			<type>1</type>
 			<locationURI>AFR_HOME/libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c</locationURI>
 		</link>
+		<link>
```

Comment:
Not sure why the iot_pkcs11_pal.c is moved, but we shall leave it and check it in.